godot-ffi/src/godot_ffi.rs

```diff
@@ -472,10 +494,48 @@ macro_rules! ffi_methods {
     };
 }
 
+/// An error representing a failure to convert some value of type `From` into the type `Into`.
+#[derive(Copy, Clone, Hash, Eq, PartialEq, Debug)]
+pub struct PrimitiveConversionError<From, Into> {
+    from: From,
+    target_ty: PhantomData<Into>,
+}
+
+impl<From, Into> PrimitiveConversionError<From, Into> {
+    pub fn new(from: From) -> Self {
+        Self {
+            from,
+            target_ty: PhantomData,
+        }
+    }
+}
+
+impl<From, Into> std::fmt::Display for PrimitiveConversionError<From, Into>
+where
+    From: std::fmt::Display + std::fmt::Debug,
+    Into: std::fmt::Display + std::fmt::Debug,
```

Comment:
Do we really need both traits here?

For error messages, the `Debug` trait tends to be more useful, as it sometimes contains information about the type that's not available otherwise (example: `Gd` with dynamic class + instance ID).

Reply:
`Error` requires both traits